python-lib/sharepoint_client.py

```diff
@@ -823,7 +823,10 @@ def get(self, url, headers=None, params=None):
         headers = headers or {}
         headers["Accept"] = DSSConstants.APPLICATION_JSON
         headers["Authorization"] = self.get_authorization_bearer()
-        return requests.get(url, headers=headers, params=params)
+        response = None
+        while not is_request_performed(response):
+            response = requests.get(url, headers=headers, params=params)
```

Comment:
If this condition `if response.status_code in [429, 503]`  in `is_request_performed` is true for a very long time, for e.g the number of requests is limited in x requests per hour, or the server is overloaded, then the loop will retry to send requests until it gets a positive response - unless I did not get how do you manage this case.  Is it possible to set a maximum number of retry ?